README.md

```diff
@@ -1,11 +1,11 @@
-# Advent of code 2020 solutions
-
-⁣    🌟  
-    🎄  
-   🎄🎄  
-  🎄🎄🎄  
- 🎄🎄🎄🎄  
-🎄🎄🎄🎄🎄  
+# Advent of code 2020 solutions <!-- omit in toc --> <!-- do not remove comment + use Markdown All in One in VSCode -->
+
+⁣    🌟\
+    🎄\
+   🎄🎄\
+  🎄🎄🎄\
+ 🎄🎄🎄🎄\
+🎄🎄🎄🎄🎄\
```

Comment:
Some IDEs (including mine) auto remove trailing spaces, it won't with trailing \